mypy/plugin.py

```diff
@@ -459,16 +459,23 @@ def lookup_fully_qualified(self, fullname: str) -> Optional[SymbolTableNode]:
         assert self._modules is not None
         return lookup_fully_qualified(fullname, self._modules)
 
-    def report_config_data(self, id: str, path: str) -> Any:
+    def report_config_data(self, id: str, path: str, is_check: bool) -> Any:
```

Comment:
Looking at the call sites maybe it is better to call this argument `context` and pass it two predefined constants like `mypy.plugin.LOAD` and `mypy.plugin.STORE`? I think this would be more user-friendly.